src/locales/strings.js

```diff
@@ -1,6 +1,6 @@
 // @flow
 
-import { getLocales } from 'react-native-localize'
+// import { getLocales } from 'react-native-localize'
```

Comment:
What is the reason for removing this and leaving it commented?

Reply:
This was only during testing, should of not gotten into the commit. Fixed.

Reply:
Didn't get updated?